.github/workflows/securityAlertsReview.yml

```diff
@@ -0,0 +1,194 @@
+name: Security Alerts Review
+
+on:
+  push:
+    paths:
+      - 'src/**.sol'
+      - ".github/workflows/securityAlertsReview.yml"
+  pull_request:
+    types:
+      - synchronize
+      - reopened
+      - ready_for_review
+  pull_request_review:
+    types:
+      - submitted
+  workflow_dispatch:
+
+jobs:
+  check-security-alerts:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Fetch PR Number
+        id: fetch_pr
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          PR_NUMBER=$(curl -s -H "Authorization: token ${GITHUB_TOKEN}" \
+            "https://api.github.com/repos/${{ github.repository }}/pulls?state=open" | jq -r '.[0].number')
```

Comment:
If I am reading this right, you are pulling all open PRs and then use the first element, is that correct?
First of all, please add comments explaining your rationales and assumptions.
Second, this seems like a very risky approach to me. How do you know which one is the latest PR and if that's the PR you are looking for?

In other git actions we use `${{ github.event.pull_request.number }}` to get the PR number but that doesnt work for every trigger event type unfortunately.

Maybe something like this would be better?
https://github.com/marketplace/actions/find-current-pull-request

Reply:
updated